### PR TITLE
pythonPackages.gym: init at 0.9.6

### PR DESCRIPTION
**Pull request by hyphon81**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/gym/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0llbhn3zdlsz2crd5grd1yygg8zp2shsclc24iqix5gw5f65clx5";
+  };
+
+  propagatedBuildInputs = [
+    numpy requests six pyglet scipy
+  ];
+
+  doCheck = false;
```
Comment (FRidh) [inline pkgs/development/python-modules/gym/default.nix]: Why are the tests disabled? Please include a comment.

File: pkgs/development/python-modules/gym/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib
```
Comment (FRidh) [inline pkgs/development/python-modules/gym/default.nix]: only lib is needed

**Review by FRidh — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/gym/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  };
+
+  propagatedBuildInputs = [
+    numpy requests six pyglet scipy
+  ];
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "The OpenAI Gym: A toolkit for developing and comparing your reinforcement learning agents.";
```
Comment (dotlambda) [inline pkgs/development/python-modules/gym/default.nix]: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

**Review by dotlambda — CHANGES_REQUESTED** [review]
Please fix the commit message typo.

File: pkgs/development/python-modules/gym/default.nix (hunk 3 above)
Comment (Ekleog) [reply pkgs/development/python-modules/gym/default.nix]: If it's about `meta.platforms`, it's already set by `buildPythonPackage` :)

**Comment by Ekleog** [comment]
(triage) @FRidh @dotlambda Looks like your comments have been addressed, can you confirm you're now OK with this?

**Review by veprbl — APPROVED** [review]

File: pkgs/development/python-modules/gym/default.nix [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+  propagatedBuildInputs = [
+    numpy requests six pyglet scipy
+  ];
+
+  # The test needs MuJoCo that is not free library.
+  doCheck = false;
+
+  meta = with lib; {
+    description = "A toolkit by OpenAI for developing and comparing your reinforcement learning agents.";
```
Comment (dotlambda) [inline pkgs/development/python-modules/gym/default.nix]: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.gym python3.pkgs.gym

**Pull request merged**